### ISSUE TITLE
ng generate component product-alerts causes "Cannot set property global of #<Object> which has only a getter" error

### ISSUE THREAD

**Issue by AdsLeech** [stackblitz/core#1752]
### **Description of Bug**
I'm going through the Angular tutorial using StackBlitz and I've got as far as "Pass data to a child component" section. As instructed, I've created a new terminal and typed in the "ng generate component product-alerts" command. After a few seconds I get the following error: 
An unhandled exception occurred: Cannot set property global of #<Object> which has only a getter
See "/tmp/ng-asC2Tb/angular-errors.log" for further details.

No components are created

### **Steps to Reproduce**

Follow the steps in the tutorial
https://angular.io/start#pass-data-to-a-child-component

### **Expected Behavior**
It's suppose to create the component files in my project

### **Screenshots/Screencast**
![22-02-2022-02-43-21](https://user-images.githubusercontent.com/100131419/155155175-9aa27f84-397d-4677-8895-b0344320d0c0.png)

**Additional Context/Questions**
I've tried searching for /tmp/ng-asC2Tb/angular-errors.log but it doesn't exist. 

Right-clicking on the app folder is supposed to allow me to create the component from there too but the Angular component option isn't listing in the menu
![22-02-2022-02-45-20](https://user-images.githubusercontent.com/100131419/155155566-6d575473-bbcf-4f96-b8a3-7393e6ebeb81.png)

**Comment by laantmik** [comment]
Totally same problem!

**Comment by RodrigoM586** [comment]
The same here.

**Comment by FDrtn1te** [comment]
same issue here...

**Comment by laantmik** [comment]
> The same here. In any case, to move forward with the project, you can manually create the folder and the corresponding files.

Yes I downloaded it and started to used Visual Studio Code after installing Angular CLI

**Comment by Benj1907** [comment]
Same issue
